### PR TITLE
Add README information for Chronolog script and fix one-sided interactions

### LINKED ISSUE #136: First interactions are one sided
In some cases, when you select a link to view a characters first interaction, it may say there isn't a first interaction to display when by definition this should not be the case.

### PR DESCRIPTION
**Pull request by ESummerscale**
(This probably should have been two separate branches but I forgot, sorry!)
Fixes #136  and #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,34 +11,90 @@ ChronoLog aims to make this easier by generating a timeline of where people are
...
+can process and visualise the text on your own machine.
+
+The following guide assumes that you are using a Unix-like machine.
+
+#### Processing Text
+
+1. Clone the ChronoLog repository to your own machine by running the following command in the terminal:  
+   `git clone https://github.com/JamesLee-Jones/ChronoLog.git`
+2. Navigate to the outermost folder, nlp-timeline-gen  
+   `cd nlp-timeline-gen`
```
Comment (BOluokun) [inline README.md:29]: I think the repository name has changed so it should be `cd Chronolog`

File: README.md [hunk 2]
```diff
@@ -11,34 +11,90 @@ ChronoLog aims to make this easier by generating a timeline of where people are
...
+    * `--sections N` splits the text into N roughly equal sections for analysis, in the case that the book is not
+      chaptered or a chapterRegex is not readily available. This defaults to `N=10`.
+    * `--narrator NARRATOR` should be supplied if the text is written in the first person. Supply the full name of the
+      narrator character.
+      For example, 'The Great Gatsby' is written in first person and the narrator is Nick Carraway, so we should
+      specify `--narrator "Nick Carraway"`</ul>
+      There are additional optional arguments for refining the analysis that can be found through the `--help` command.
+5. Run the script using the appropriate arguments for your text. In our example, this
+   is  
+   `python ./chronolog.py --chapterRegex numeral --narrator "Nick Carraway texts/the_great_gatsby.txt`.
```
Comment (BOluokun) [inline README.md:53]: The closing quote is missing, should be `--narrator "Nick Carraway"`

File: README.md [hunk 3]
```diff
@@ -11,34 +11,90 @@ ChronoLog aims to make this easier by generating a timeline of where people are
...
+      chaptered or a chapterRegex is not readily available. This defaults to `N=10`.
+    * `--narrator NARRATOR` should be supplied if the text is written in the first person. Supply the full name of the
+      narrator character.
+      For example, 'The Great Gatsby' is written in first person and the narrator is Nick Carraway, so we should
+      specify `--narrator "Nick Carraway"`</ul>
+      There are additional optional arguments for refining the analysis that can be found through the `--help` command.
+5. Run the script using the appropriate arguments for your text. In our example, this
+   is  
+   `python ./chronolog.py --chapterRegex numeral --narrator "Nick Carraway texts/the_great_gatsby.txt`.  
+   Note the expected order: `python ./chronolog.py [optional arguments] [filename]`
```
Comment (BOluokun) [inline README.md:54]: I actually put the filename first when I use the tool and it still works.

**Review by BOluokun — CHANGES_REQUESTED** [review]
Good, just a few suggestions.

File: README.md (hunk 3 above)
Comment (ESummerscale) [reply README.md]: Thank you! These and the other issues you pointed out should be fixed now :)

**Review by BOluokun — APPROVED** [review]
LGTM!

**Pull request merged**